main.js

```diff
@@ -26,136 +26,99 @@ categoryBoxWrapper.addEventListener('click', activateCategory);
 startActivityButton.addEventListener('click', submitForm);
 
 //Add functions below
+function show(element) {
+  element.classList.remove('hidden');
+}
+
+function hide(element) {
+  element.classList.add('hidden');
+}
+
 function submitForm() {
   event.preventDefault();
   if (checkInputValidity()) {
     createNewActivity();
-  };
-};
+    hide(activityInputForm);
+    show(timerBoxWrapper);
+  }
+}
 
 function checkInputValidity() {
   //Add validation conditional here for activity boxes to make sure one is clicked
-  if (!accomplishInput.value === true) {
+  if (!accomplishInput.value) {
```

Comment:
Nice. Jessica and I discussed and she tried but couldn't get it to work without strictly equals. Good deal!

Reply:
Nice catch here @knees4bees! I think I was trying `=== false` at first, then switched to true to get the functionality I needed, but didn't realize that the `=== true` was now redundant.